src/Plugin.php

```diff
@@ -65,27 +65,15 @@ public static function init() {
 
     add_shortcode('user-login-form', __NAMESPACE__ . '\UserLoginForm::getOutput');
 
-    // Disable self pingbacks.
-    add_action('pre_ping', __CLASS__ . '::pre_ping');
-
-    // Disable pingback functionality and remove x-pingback HTTP header.
-    add_filter('xmlrpc_methods', __CLASS__ . '::xmlrpc_methods');
-    add_filter('wp_headers', __CLASS__ . '::wp_headers');
-
     // Disables admin discussion options for pingbacks.
-    add_filter('pre_update_default_ping_status', '__return_false');
     add_filter('pre_option_default_ping_status', '__return_zero');
-    add_filter('pre_update_default_pingback_flag', '__return_false');
     add_filter('pre_option_default_pingback_flag', '__return_zero');
 
     // Disables insertion of rel="pingback" tags.
-    add_filter('bloginfo_url', __CLASS__ . '::bloginfo_url', 10, 2);
-
-    // Unchecks 'allow trackbacks and pingbacks' in the Discussion meta box of post editing page.
-    add_action('post_comment_status_meta_box-options', __CLASS__ . '::post_comment_status_meta_box_options');
+    add_filter('pings_open', '__return_false', 20, 2);
```

Comment:
1. I'm fairly sure `__return_false` takes only zero to one arguments.

1. The priority could be a bit higher, too; e.g. 100.

Reply:
Ouch, you are right.